src/test/java/core/basesyntax/transaction/FruitTransactionTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.transaction;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    private final FruitTransaction transaction = new FruitTransaction(null, null, 0);
```

Comment:
we don't test models explicitly, they are considered tested when we test services that use them